addons/escoria-core/game/escoria.gd

```diff
@@ -15,7 +15,13 @@ const ESCORIA_CORE_PLUGIN_NAME: String = "escoria-core"
 onready var main = $main
 
 
+func _on_game_is_loading():
+	print("GAME IS LOADING")
```

Comment:
Consider changing the `print` statement to a logger statement?

Reply:
Done